cost-analyzer/templates/federator-pod-template.yaml

```diff
@@ -0,0 +1,66 @@
+{{- if .Values.federatedETL.federator }}
+{{- if .Values.federatedETL.federator.enabled }}
+apiVersion: v1
+kind: Pod
```

Comment:
@nickcurie In most cases, we don't want to be directly deploying a pod to a k8s cluster. Instead, we want to be creating a [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) which will manage a set of pods.

The pod spec you have here is good, but I'd recommend you wrapping a Deployment spec around this pod spec. You can use the [QSR template](https://github.com/kubecost/cost-analyzer-helm-chart/blob/v1.104/cost-analyzer/templates/query-service-deployment-template.yaml) as reference, but instead of a StatefulSet I'd recommend a Deployment!

cc @kaelanspatel @jessegoodier @srpomeroy Thoughts?

Reply:
Prefer not to have uncontrolled workloads. I agree; Deployment likely makes the most sense.